src/test/java/uk/gov/hmcts/darts/retention/service/impl/RetentionPostServiceImplTest.java

```diff
@@ -130,7 +130,7 @@ void fail_NoCase() {
 
         var exception = assertThrows(
```

Comment:
shall we add a test where we verify that if the query param is null it defaults to `false` as per the specification?

Reply:
scenario now added